source/nodejs/adaptivecards/src/utils.ts

```diff
@@ -4,6 +4,25 @@ import * as Enums from "./enums";
 import * as Shared from "./shared";
 import { HostConfig } from "./host-config";
 
+export function formatString(s: string, ...args: any[]): string {
```

Comment:
would this perhaps be a little more discoverable if it were an extension method on string?

Reply:
The recommendation in the wild is to not extend string to avoid conflicts.